rancher-common/README.md

```diff
@@ -31,7 +31,8 @@ Kubernetes version to use for Rancher server RKE cluster
 - Default: **`"0.12.0"`**
 Version of cert-mananger to install alongside Rancher (format: `0.0.0`)
 
-Used in the case of older Rancher versions.
```

Comment:
"Used in the case of older Rancher versions." can be removed I think. The variable is always used.

Reply:
Agreed, was originally there to justify its existence as opposed to hard-coding all versions, seems reasonable to remove it.